# My Jupyter Notebook on IBM Watson Studio


### Osmi
### Business Analyst

### *I am interested in data science because we need to analyze historical data and predict new cases to take important actions.*

### Multiplying two big number

In [2]:
multiply = 12412412*123123124
print(multiply)

1528254941815088


*** 

output = 1528254941815088 

***
Our program website can be found at <a href="http://earthdatascience.org" target="_blank">this link</a>.

`Here is some code!`